src/main/java/org/commcare/modern/database/IndexedFixturePathsConstants.java

```diff
@@ -1,28 +0,0 @@
-package org.commcare.modern.database;
-
-/**
- * @author Phillip Mates (pmates@dimagi.com)
- */
-public class IndexedFixturePathsConstants {
```

Comment:
I think FormPlayer also uses this data, so it's probably not possible to move it to Android